src/components/MultiBarToggle.js

```diff
@@ -11,6 +11,10 @@ const DEFAULT_ACTION_STAGING_DURATION = 100;
 const DEFAULT_ACTION_ANIMATION_DURATION = 200;
 const DEFAULT_NAVIGATION_DELAY = 500;
 const DEFAULT_EXPANDING_ANGLE = 135;
+const DEFAULT_OVERLAY_ACTIVE = false;
+
+const width = Dimensions.get('window').width;
```

Comment:
Extract width and height in one line:
`const { width, height } = Dimensions.get('window');`

Reply:
fixed!